.github/workflows/build-and-test.yml

```diff
@@ -265,6 +265,12 @@ jobs:
         name: libopencm3-stm32f4xx-build-artifact
         path: build-libopencm3/samples
 
+    - name: Download crt0.s build artifact for STM32F4xx
```

Comment:
NIT: For building, uploading and deleting we use the order "CMSIS, crt0.s, libopencm3". Can please do it in the same manner for downloading?

Reply:
Uhh, we could have done the same in the robots file. Will do so in a follow up if I really care enough ;)